debezium-server/debezium-server-ybexporter/src/main/java/io/debezium/server/ybexporter/ExportStatus.java

```diff
@@ -300,6 +305,30 @@ public void updateQueueSegmentMetaInfo(long segmentNo, long committedSize, Map<P
 
     }
 
+    public boolean checkIfSwitchOperationRequested(String operation) throws SQLException {
+        Statement selectStmt = metadataDBConn.createStatement();
+        String query = String.format("SELECT json_text from %s where key = '%s'",
+            JSON_OBJECTS_TABLE_NAME, MIGRATION_STATUS_KEY);
+        try {
+            ResultSet rs = selectStmt.executeQuery(query);
+            while (rs.next()) {
+                JSONObject record = new JSONObject(rs.getString("json_text"));
+                if (record.get("CutoverRequested").toString().equals("true")
```

Comment:
nit: probably more readable to have a switch on operation here, and then check for the appropriate msr field accordingly.

Reply:
done